client/js/content.js

```diff
@@ -0,0 +1,26 @@
+var images = document.getElementsByTagName('img');
+// for (var i = 0, l = images.length; i < l; i++) {
+//   images[i].src = 'http://placekitten.com/' + images[i].width + '/' + images[i].height;
+// }
+
+var textbox = document.getElementsByClassName("episode")
+for (var i = 0, l = textbox.length; i<l; i++){
+    console.log(textbox[i])
+    var imdbRatings = document.createElement('div')
+    imdbRatings.innerHTML = "IMDB Ratings : 5.0"
+    imdbRatings.style.width = '100%';
```

Comment:
You can add a class to the div and not add individual style property. This will help in creative multiple style with less redundancy